### ISSUE TITLE
Add support for twitter:image

### ISSUE THREAD

**Issue by edazpotato** [jonasmerlin/astro-seo#30]
And the other relate attributes like `twitter:image:alt`. I know this isn't necessary because most platforms use opengraph and twitter interchangeably, but it's important for completions sake.

**Comment by jonasmerlin** [comment]
@edazpotato Yeah, I thought about this while implementing it and decided to go with the minimal set for now. But I don't have strong feelings about this one way or the other, so we can add it. In the meantime, if you already want to set it, you can use `extend.meta` to do so

**Comment by edazpotato** [comment]
Thanks! I'll use `extend.meta` until this gets implemented

**Comment by FlipFloop** [comment]
Any updates on this?
